portal-impl/src/com/liferay/portal/service/impl/UserLocalServiceImpl.java

```diff
@@ -3495,6 +3495,29 @@ public List<User> searchBySocial(
 			).build(),
 			start, end, orderByComparator);
 	}
+	
+	@Override
+	public List<User> searchBySocial(
```

Comment:
Is there a specific reason to have a different function instead of modify the one before this to accept the new userGroupIds param?

Reply:
Hi @smottal 
There are a lot of usages of this method in other modules such as change-tracking, dynamic-data-mapping and mentions, I thought I couldn't modify them because it was outside the scope of the customer original issue